### ISSUE TITLE
Android Kotlin: 08.1 Intent Types

### ISSUE THREAD

**Issue by shubhmittal07** [girlscript/winter-of-contributing#3048]
<hr>

## Description :scroll:

* Give a step-by-step tutorial of Intent and it's types in Android (Kotlin).
*  Required Documentation or Video or Audio.

<hr>

## Domain of Contribution :bar_chart:

- [x] Android Dev (Kotlin)

<hr>
## Please Read it Carefully Before Contributing
* Changes should be made in Android_Development_With_Kotlin/ directory under Android_Development_With_Kotlin branch.
* Inside Android_Development_With_Kotlin/ directory creates a directory and names it according to the topic ( for this issue 09. Dynamic User Interface)
* Inside 09. Dynamic User Interface/ directory add your task with subtopic number and name and make changes accordingly.
* You have to push your PR to the Android_Development_With_Kotlin branch only.
* You can take help to the references but don't simply try to copy it. Just explain in your own word.
* This issue is for GWOC'21 contributors only.
* You need to push in a single commit. If there exists more than one commit then you need to squash your commits into one commit.
* Will assign you on a first come first serve basis.
* Please try to complete your assigned task within 7 days. If you need some more time then, please inform mentors and supervisors.
* If you have two previously assigned issues on Android-Kotlin pending then please first complete them.
> Supervisor @rahulsain

**Comment by shubhmittal07** [comment]
I would be contributing in the Documentation part

**Comment by shubhmittal07** [comment]
Can we also contribute projects , it would be like a tutorial. @rahulsain

**Comment by rahulsain** [comment]
Sure, please provide me with what project you are currently thinking of implementing?

**Comment by shubhmittal07** [comment]
I was thinking of a project which deals with the implementation of Recycler View . It will be a project for beginners. @rahulsain

**Comment by rahulsain** [comment]
Recycler View and API?
we can create an app which display super power of any favorite franchise like marvel etc,

**Comment by shubhmittal07** [comment]
Yeah I can do that. Will start working on it as soon as possible.
Activity 1 - >will include , multiple franchise to select like Marvel , DC etc.
Activity 2  -> will include the super power of the selected franchise
@rahulsain

**Comment by rahulsain** [comment]
> Yeah I can do that. Will start working on it as soon as possible. Activity 1 - >will include , multiple franchise to select like Marvel , DC etc. Activity 2 -> will include the super power of the selected franchise @rahulsain

Sure, but make sure that you write clean and good quality code

**Comment by shubhmittal07** [comment]
Yeah I will make sure of it
